software/passbolt.yml

```
@@ -1,6 +1,6 @@
 name: Passbolt
 website_url: https://www.passbolt.com/
-description: Password manager dedicated for managing passwords in a collaborative way on any Web server, using a MySQL database backend.
+description: Manage passwords in a collaborative way on any Web server, using a MySQL database backend.
```

```suggestion
description: Collaborative password manager.
```

Shorter, cleaner description